pkg/cmd/server/kubernetes/master.go

```diff
@@ -11,6 +11,7 @@ import (
 
 	kapi "k8s.io/kubernetes/pkg/api"
 	"k8s.io/kubernetes/pkg/client/record"
+	osclient "github.com/openshift/origin/pkg/client"
```

Comment:
> This dependency looks weird. I think you need two controllers, one for origin and one for kube. Otherwise, HPA won't work for DCs in the remote kube case.

@smarterclayton Confirmed that HPA can be broken for external kubernetes cases for now.  I'll open an issue upstream to discuss how to fix this.